docs/data-guidelines.md

```diff
@@ -280,3 +280,23 @@ Members of this mixin are available to `HTMLAnchorElement` and `HTMLAreaElement`
    ```
 
 This guideline was proposed in [#8929](https://github.com/mdn/browser-compat-data/issues/8929), based in part on previous discussion in [#472](https://github.com/mdn/browser-compat-data/issues/472).
+
+## Choosing an experimental status
+
+Generally, when a feature is supported by one and only one browser engine, set `experimental` to `true`. When a feature is supported by two or more engines, then set `experimental` to `false`. Some exceptions apply, however, for long-standing features and features behind flags and prefixes.
+
+If a feature is supported behind flags only, no matter how many engines, then set `experimental` to `true`.
+
+If a feature is supported behind incompatible prefixes only (such as `-webkit-` in one engine and `-moz-` in another), no matter how many engines support the feature overall, then set `experimental` to `true`. If two or more engines support a feature behind a common prefix (such as `-webkit-` only), then set `experimental` to `false`.
+
+If a feature has been supported by one and only one engine without major changes for two or more years (relative to the most-recent browser release introducing support for the feature, or the most-recent browser release since the last major change) , then `experimental` may be set to `false`.
```

Comment:
OK, this is the sunsetting part of the guideline. It started out as this:

> If a feature has been supported by one and only one engine for two or more years without major changes, then set `experimental` to `false`.

But I had a bit of trouble with it. We have release dates for browser versions, not engines. So the new text contains quite the parenthetical to address that. It's trying to capture when features achieve stasis, if not consensus. It's still awkward for reviews and authors (you have to look at all the release dates to figure out what the right value is), but it's answerable and might accommodate oddities (like Edge supporting a thing before Chrome) that going with least-recent release wouldn't.

Reply:
OK, this is the good stuff that we need to get right.

I think your original attempt was pretty good, but I see the problem. Throughout this guideline, one could understand "engine" as "the set of browsers using the same engine" and it mostly works. The question is which date to use here. I think it should actually be least-recent, because there is a high variability in when a Chromium version rolls out to all of Chrome (+Android), Edge, Opera (+Android), Samsung Internet and WebView. I think Opera for Android or Samsung Internet will tend to be the last.

I would propose:

> Otherwise, if a feature has been supported by some browser for two or more years without major changes and there is no support behind a flag in some other browser with major differences from the first, then set `experimental` to `false`.

The "Otherwise" is to not have to also carve out the preceding cases by saying "one and only one" in some way.